db/migrations/2303281630_fix_before_update_items_trigger.sql

```diff
@@ -0,0 +1,38 @@
+-- +migrate Up
+
+DROP TRIGGER `before_update_items`;
+-- +migrate StatementBegin
+CREATE TRIGGER `before_update_items` BEFORE UPDATE ON `items` FOR EACH ROW
+BEGIN
+  IF NOT OLD.url <=> NEW.url THEN
+    IF NEW.url IS NOT NULL THEN
+      SET @platformID = (SELECT platforms.id FROM platforms WHERE NEW.url REGEXP platforms.regexp ORDER BY platforms.priority DESC LIMIT 1);
+
+      SET NEW.platform_id=@platformID;
+    ELSE
+      SET NEW.platform_id = NULL;
+    END IF;
```

Comment:
why this change? Isn't it just another way to write the same think in SQL?

Reply:
probably you don't not the `@platformID` local var anymore (but not really important)